riscv/decode.h

```diff
@@ -395,20 +395,20 @@ inline long double to_f(float128_t f){long double r; memcpy(&r, &f, sizeof(r));
 #define e512 512  // 512b elements
 #define e1024 1024  // 1024b elements
 
-#define vsext(x, sew) (((sreg_t)(x) << (64-sew)) >> (64-sew))
-#define vzext(x, sew) (((reg_t)(x) << (64-sew)) >> (64-sew))
+#define vsext(x, sew) (((sreg_t)(x) << (64 - sew)) >> (64 - sew))
+#define vzext(x, sew) (((reg_t)(x) << (64 - sew)) >> (64 - sew))
 
 #define DEBUG_RVV 0
 
 #if DEBUG_RVV
 #define DEBUG_RVV_FP_VV \
-  printf("vfp(%lu) vd=%f vs1=%f vs2=%f\n", i, to_f(vd), to_f(vs1), to_f(vs2));
+  printf("vfp(%lu) vd=%f vs1=%f vs2=%f \n", i, to_f(vd), to_f(vs1), to_f(vs2));
 #define DEBUG_RVV_FP_VF \
-  printf("vfp(%lu) vd=%f vs1=%f vs2=%f\n", i, to_f(vd), to_f(rs1), to_f(vs2));
+  printf("vfp(%lu) vd=%f vs1=%f vs2=%f \n", i, to_f(vd), to_f(rs1), to_f(vs2));
 #define DEBUG_RVV_FMA_VV \
-  printf("vfma(%lu) vd=%f vs1=%f vs2=%f vd_old=%f\n", i, to_f(vd), to_f(vs1), to_f(vs2), to_f(vd_old));
+  printf("vfma(%lu) vd=%f vs1=%f vs2=%f vd_old=%f \n", i, to_f(vd), to_f(vs1), to_f(vs2), to_f(vd_old));
 #define DEBUG_RVV_FMA_VF \
-  printf("vfma(%lu) vd=%f vs1=%f vs2=%f vd_old=%f\n", i, to_f(vd), to_f(rs1), to_f(vs2), to_f(vd_old));
+  printf("vfma(%lu) vd=%f vs1=%f vs2=%f vd_old=%f \n", i, to_f(vd), to_f(rs1), to_f(vs2), to_f(vd_old));
```

Comment:
These aren't style changes - they're changing what gets printed!

Reply:
Yeah, they were replaced when I tried to add space before '\'. I'll restore them later.